src/proxy/main.c

```diff
@@ -33,7 +34,8 @@ static void usage(char *argv[]) {
         printf("Usage: %s \n"
                "\t[-h help]\t\t Print this help message.\n"
                "\t<operation>\t\t Required. Operation to perform. Must be one of [%s,%s].\n"
-               "\t<node>_<service>\t Required. Input the name of the node the required service is supposed to run on.\n",
+               "\t<node>_<service>\t Required. Input the name of the node the required service is supposed to run on.\n"
+               "\t<[-v version]\t Get current version of hirte-proxy\n",
```

Comment:
Hmm, doesn't above parameter description implies, that you need to specify the version after `-v` option? Shouldn't the description be writter as?

```
"\t<[-v]\t Print version of hirte-proxy\n",
```

Reply:
As you can see there the line description for the help command is also written as `[-h help]`.
I can change both to something like [-v|version] or [-v|--version] if you think it's better.

Reply:
OK, let's leave it as is now, we can improve in a different PR